.github/workflows/django-ory-cloud.yml

```
@@ -0,0 +1,31 @@
+name: E2E Tests for django-ory-cloud
```

The status badge will show this text, so it should be as short as possible.

```suggestion
name: django-ory-cloud
```